compiler/pash_runtime.py

```diff
@@ -686,12 +688,19 @@ def add_eager(eager_input_id, graph, fileIdGen, intermediateFileIdGen, use_dgsh_
     new_id = new_fid.get_ident()
 
     if use_dgsh_tee:
+        assert(False)
```

Comment:
This needs to be tested and done because it will soon become the default instead of `eager` (it is even easier than `eager` since it is not given an intermediate FileId if I am not mistaken). You can configure pash to add this using the `--dgsh_tee` flag.

Reply:
I added this to the `TODO`-list.